### PR TITLE
Review of Lola's WebRTC tutorial

### PR DESCRIPTION
**Pull request by chrisdavidmills**

### CODE REVIEW

**Comment by chrisdavidmills** [comment]
Hi @lolaodelola ! I've reviewed your tutorial series, and made some changes — see this PR!

In short, I think it looks amazing, and I really enjoyed working through it. I've not changed an awful lot of content; I've mainly just got it into MDN style and tweaked things a bit.

I'll send you some accompanying comments over mail too. I think it would be best for you to review the changes as they are here first, and make sure you are happy with them. After that we can perhaps discuss the other changes.

Thanks again for such great work!

**Comment by Rumyra** [comment]
Awesome - I'll put aside some time for this which isn't just a 5 min email check ☺️

Quite looking forward to doing some webRTC 🎉

**Comment by chrisdavidmills** [comment]
I actually ended up removing you as a reviewer @Rumyra , sorry for the confusion — you were added by the codeowners system.

That said, I am happy for you to check it out too. I've checked the language pretty thoroughly, but I'd be happy for you to look and code and see if you can suggest any improvements.

File: files/en-us/web/api/webrtc_api/build_a_phone_with_peerjs/connect_peers/create_a_peer_connection/index.html [hunk 1]
```diff
@@ -22,14 +22,17 @@
   <li>
 <p>Using the peerJS framework, you'll want to connect the <code>localPeer</code> to the <code>remotePeer</code>. PeerJS gives us the <code>connect()</code> function, which takes a peer ID to connect to. Add this block below your previous code:</p>
 
-<pre class="brush: js">function connectPeers() {
-    peer.connect(code);
-}</pre>
+<pre class="brush: js">
+    let conn;
```
Comment (Ryuno-Ki) [inline files/en-us/web/api/webrtc_api/build_a_phone_with_peerjs/connect_peers/create_a_peer_connection/index.html:26]: Before, we had the first line on the opening element line.

File: files/en-us/web/api/webrtc_api/build_a_phone_with_peerjs/connect_peers/create_a_peer_connection/index.html [hunk 2]
```diff
@@ -22,14 +22,17 @@
...
+    }
+</pre>
   </li>
   <li>
 <p>When a connection is created, let's use the PeerJS framework's <code>on(‘connection')</code> to set the remote peer’s ID and open the connection. The function for this listener accepts a <code>connection</code> object which is an instance of the <code>DataConnection</code> object (which is a wrapper around WebRTC’s <a href="/en-US/docs/Web/API/RTCDataChannel"><code>RTCDataChannel</code></a>); within this function you'll want to assign it to a variable. Again you’ll want to create the variable outside of the function so that you can assign it later. Add the following below your previous code:</p>
 
-<pre class="brush: js">let conn;
+<pre class="brush: js">
 peer.on('connection', function(connection){
     conn = connection;
```
Comment (Ryuno-Ki) [inline files/en-us/web/api/webrtc_api/build_a_phone_with_peerjs/connect_peers/create_a_peer_connection/index.html:37]: What happens to this now?

File: files/en-us/web/api/webrtc_api/build_a_phone_with_peerjs/connect_peers/create_a_peer_connection/index.html (hunk 1 above)
Comment (chrisdavidmills) [reply files/en-us/web/api/webrtc_api/build_a_phone_with_peerjs/connect_peers/create_a_peer_connection/index.html:26]: I think this is ok

File: files/en-us/web/api/webrtc_api/build_a_phone_with_peerjs/connect_peers/create_a_peer_connection/index.html (hunk 2 above)
Comment (chrisdavidmills) [reply files/en-us/web/api/webrtc_api/build_a_phone_with_peerjs/connect_peers/create_a_peer_connection/index.html:37]: this is lola's fix for the bug we previously had

**Comment by chrisdavidmills** [comment]
OK, looks good, thanks for the updates @lolaodelola.

I'm going to merge this now; if any other fixes need making, let's iterate in a future PR.

**Pull request merged**